content/java/concepts/math-methods/terms/multiplyExact/multiplyExact.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.multiplyExact()'
+Description: 'Returns the product of its arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Function'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.multiplyExact()`** method returns the product of its arguments. It is a static method and it is accessed by using the `Math` class.
+
+## Syntax
+
+```pseudo
+Math.multiplyExact(x, y)
+```
+
+The `.multiplyExact()` method takes two parameters. `x` and `y` (a multiplicand and multiplier). Both parameters must be of either [type](https://www.codecademy.com/resources/docs/java/data-types) `int` or `long`.
```

Comment:
```suggestion
The `.multiplyExact()` method takes two parameters:`x` and `y` (a multiplicand and multiplier). Both parameters must be of either [type](https://www.codecademy.com/resources/docs/java/data-types) `int` or `long`.
```

Looking again, colon works better here. Was mainly switching multiplicand and multiplier to match x and y, respectively